### ISSUE TITLE
Need help identifying manufacturer for new model

### ISSUE THREAD

**Issue by dinesh-58** [833M0L3/NETTV#2]
Hi, how did you find out that the manufacturer was Home Digital? This method works on our old settop box but MI remote (with Home Digital selected) doesn't respond on a newer box.

**Comment by dinesh-58** [comment]
Both have model no. GX-1000 on the bottom & the physical remote for the old one works with the new one as well

**Comment by 833M0L3** [comment]
@dinesh-58 Could you check the firmware version of your NETTV Set up box? It seems like the latest one has patched this method.

**Comment by 833M0L3** [comment]
@dinesh-58 Yes , that's the firmware version. Seems like they patched it on the 20809.

**Comment by dinesh-58** [comment]
That sucks. Do you know the exact chipset used in these? Or should I just open up the box? I found one guy that makes custom roms for these type of boxes. https://aidanrom.com/

**Comment by 833M0L3** [comment]
@dinesh-58 The white box runs on Amlogic S905X.

If you want to bypass the app install restrictions, then there is another method.

You could open the setupbox and use it's UART pin to directly get the shell. And using the shell you can install any apk. I did it using that method. 
![MVIMG_20230613_125744.jpg](https://github.com/833M0L3/NETTV/assets/59522309/acab884a-c425-4c2d-9653-174513eba30f)


![MVIMG_20230613_124923.jpg](https://github.com/833M0L3/NETTV/assets/59522309/7b0bc0ba-98e5-4e69-a766-405dddeda895)

**Comment by 833M0L3** [comment]
@dinesh-58 You could also try downgrading the firmware. I have the firmware of version 020F07. I downloaded it through NETTV's OTA download link when it was being pushed to the masses. 

[020F07.zip](https://gofile.io/d/PoaJSN)

**Comment by dinesh-58** [comment]
Yoo thanks man. I'll try these out some time and report back

**Comment by Subas78** [comment]
Mine is the older version but there is no Developers Option to enable usb debugging, it tried without it and it says adb no connection could be made because the target machine actively refused it. Please help

**Comment by Subas78** [comment]
Im not a tech guy, i am related to management field, the 2nd option looks risky

**Comment by dinesh-58** [comment]
@Subas78 if you were able to get into android settings using MI remote, follow these steps to enable developer settings:

1. In android settings, scroll to the bottom and go to `About media box`
2. Find build number and press ok button around 7 times (until it says that you are a developer)
3. Now if you go back, you will find developer settings and can enable usb debugging there

**Comment by Subas78** [comment]
Thanks, solved

**Comment by 833M0L3** [comment]
@Subas78 What was the issue?

**Comment by Subas78** [comment]
USB debugging is solved, and i have installed, that launcher as well, but i don't know why it finds it harder to install heavy size apk such as aptoid TV and Es file explorer.

**Comment by 833M0L3** [comment]
@Subas78 Harder as in , it's taking bit more time to install ?

If that's the case then make sure your device has a good wifi coverage or Ethernet cable is good if you are using one. Big apk takes time as it has be transferred to the TV box through the network and takes a lot of bandwidth.

**Comment by Subas78** [comment]
It failed during the installation process. Yea you're right it must be because slow Internet connection i will try with Ethernet cable. Thanks a lot for the support. It worked, am sure this problem will be solved.

**Comment by 833M0L3** [comment]
@Subas78 Could you share the error message of the adb when the installation fails?

**Comment by Subas78** [comment]
Yea, sure I'll that try again on  tomorrow morning.

**Comment by Subas78** [comment]
Thank you. It worked

**Comment by rdm222** [comment]
Hi sir Plz give me the firmware of version 020F07.

**Comment by rdm222** [comment]
Above link is not working sir

**Comment by dinesh-58** [comment]
@rdm222 https://gofile.io/d/3KIRWN

**Comment by rdm222** [comment]
Thank u sir.

**Comment by 833M0L3** [comment]
@Ultron96 here you go 

![IMG_20230613_130554](https://github.com/833M0L3/NETTV/assets/59522309/20aa9ac8-a8d0-40ba-a54e-09dc73ae694f)

**Comment by Ultron96** [comment]
Thank you so much

**Comment by Ultron96** [comment]
@833M0L3 The gap between the arrows is confusing me, the pin right next to gnd is vcc or rxd ?

**Comment by pranavkafle** [comment]
@833M0L3 [020F07.zip](https://gofile.io/d/PoaJSN) is no longer available. Could you upload it in main? Thanks

**Comment by dinesh-58** [comment]
@pranavkafle [Here you go](https://drive.google.com/file/d/1YASHnrl-q03KtfWWX2JjHjO17dRSq4Tu/view?usp=sharing)

**Comment by pranavkafle** [comment]
@833M0L3 , I am new to serial communication but I wanted to try and I purchased a TTL-USB converter and connected the jumper cable like the following using the device pin map you shared.

GND (Device) -> GND (USB to TTL)
TXD (Device) -> RXD (USB to TTL)
RXD (Device) -> TXD (USB to TTL)

I am unable to get any response in PuTTY. My USB is connected to COM3. When I don't connect GND, I get back some garbled text but when I connected GND as well, there is no response at all.

Could you provide a guide on what parameters you put in PuTTY— baud, parity, bits, stops. How did you manage to open shell and then connect with adb to install apps?

Thanks in advance!

**Comment by dinesh-58** [comment]
@pranavkafle Hi could you open a new issue for this? Github issues are meant for figuring on just one problem. Also, everyone who has ever left a comment here gets notified about your comment

**Comment by 833M0L3** [comment]
@pranavkafle watch this video to have a deeper understanding of what's happening https://youtu.be/01mw0oTHwxg

For your case, I am sure it probably has to do with the bad bit rate, it's either 112500 or 9600, I also forgot. Give both a try and update us here on which worked.

**Comment by krishnastunning** [comment]
cannot connect to 192.168.1.80:5555: No connection could be made because the target machine actively refused it. (10061)

**Comment by 833M0L3** [comment]
@krishnastunning make sure your USB Debugging mode is turned on and when you do connect from cmd , a pop up box is shown on the NETTV box , you must tick always and and click on allow. Also , make sure the box and the device is on same network.

**Comment by dinesh-58** [comment]
For any future visitors, please refer to the [new website](https://833m0l3.github.io/NETTV/) for info on settop box models and new methods to enable developer settings. 

**Please open a [new issue](https://github.com/833M0L3/NETTV/issues/new) if your problem is unrelated to firmware version 020809**
